mailbox/store/src/test/java/org/apache/james/mailbox/store/search/AbstractMessageSearchIndexTest.java

```diff
@@ -1143,4 +1146,30 @@ public void searchShouldReturnEmptyWhenMailboxBelongingToAnotherMailboxSession()
 
         assertThat(messageSearchIndex.search(otherSession, mailbox, searchQuery)).isEmpty();
     }
+
+    @Test
+    public void searchShouldOrderByInternalDateWhenSortOnSentDateAndNoCorrespondingHeader() throws Exception {
```

Comment:
I don't agree, we are searching. Then it should be the first verb.

In ES there is no sentDateComparator. It is then pointless.

We already have a test with All SendDate. We just miss the Mixed test.